### PR TITLE
Adds docs about alerts data in Anomaly Explorer

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds documentation about the alerts data integration in the Anomaly Explorer.

### Preview

Screenshot 2023-11-22 at 14 22 46

### CODE REVIEW

File: docs/en/stack/ml/anomaly-detection/ml-buckets.asciidoc [hunk 1]
```diff
@@ -43,6 +43,20 @@ typical and actual values, and probability. The **Anomaly explanation** section
 helps you to interpret a given anomaly by providing further insights about its 
 type, impact, and score.
 
+If you have <<creating-anomaly-alert-rules>> applied to an {anomaly-job} and an 
+alert is triggered for the rule, you can view how the alert correlates with the 
+{anomaly-detect} results in the **Anomaly Explorer** by using the 
+**Anomaly timeline** swimlane and the **Alerts** table. The **Alerts** table
```
Comment (darnautov) [inline docs/en/stack/ml/anomaly-detection/ml-buckets.asciidoc:49]: should we refer to it as `panel`?
```suggestion
**Anomaly timeline** swimlane and the **Alerts** panel. The **Alerts** panel 
```

**Review by darnautov — COMMENTED** [review]
I recon we should also mention that Alerts panel contains total number of active, recovered and untracked alerts for the current time range and job selection. 

And the swim lane context menu contains the counters for selected time buckets.

**Comment by szabosteve** [comment]
@darnautov Updated the PR based on your comment via [558a09b](https://github.com/elastic/stack-docs/pull/2598/commits/558a09b469f7a0dc65eb6eedc5fb94e63f704d8b).

**Review by darnautov — APPROVED** [review]
LGTM

**Comment by szabosteve** [comment]
@elasticsearchmachine test this please

**Pull request merged**